### PR TITLE
Allow for alternative verbiages for login link email

### PR DESCRIPTION
**Pull request by Shinsina**
mundo EMAIL:
Screenshot from 2023-03-13 14-45-35

Packworld Email:
Screenshot from 2023-03-07 12-49-20

Used https://www.geeksforgeeks.org/mongoose-findbyid-function/ as a reference for Mongoose's `findById` method to determine that applicationId should be a string.

EDIT: Additional screenshots showing UI updates to allow for language options:

Screenshot from 2023-04-13 11-01-23
Screenshot from 2023-04-13 10-10-05
Screenshot from 2023-04-13 10-15-37
Screenshot from 2023-04-13 10-16-08

### CODE REVIEW

**Comment by Shinsina** [comment]
[Translate Newsletter Sign-up Confirmation verbiage](https://trello.com/c/mTiOcAqU/493-translate-newsletter-sign-up-confirmation-verbiage)

**Review by solocommand — CHANGES_REQUESTED** [review]
@Shinsina a few changes here. First off, the indentation shouldn't be changed for the plain text, as this will be treated literally and indented weird when delivered.

Rather than hardcoding an ID, add a language selector to the app settings screen https://identity-x.parameter1.com/orgs/5e28a144f7614f1d488bd70e/apps/edit/5e28a4c858e67b86c955ae4d. Default the setting to `en-us` in the application model's schema def, and allow `es-mx` as an enum option. You should be able to swap between languages via the UI here and test login links.

Store these templates as different files (such as `./templates/en-us.js`). Pass the required variables through to them and return the text/html/subject. Something like this:
```js
module.exports = ({
  sendingDomain,
  supportEmail,
  url,
  appName,
  addressValues,
} = {}) => {
  html: `...template`,
  text: `...template`,
  subject: `...template`,
};
```

File: services/application/src/actions/user/send-login-link.js [hunk 1]
```diff
@@ -54,57 +56,39 @@ module.exports = async ({
...
+      supportEmail,
+      url,
+      appName,
+      addressValues,
+    }),
+  };
+
+  const templateForLanguage = templateByLanguage[language];
+  if (!templateForLanguage) throw createError(404, `No template available for language: ${language}!`);
+  const { subject, html, text } = templateForLanguage;
```
Comment (solocommand) [inline services/application/src/actions/user/send-login-link.js]: Lets clean this part up a bit -- create an `email-templates/login-link/index.js` file that imports and exports the language functions.

Then this whole section can be condensed to something like this:
```js
if (!template[language]) throw createError(404, ...);
const { subject, html, text } = template[language]({ sendingDomain, ... });
```

**Review by solocommand — CHANGES_REQUESTED** [review]
Looks good, one cleanup change to make. 

I'm also not sure that this _should_ throw an error if there is no language template...IMO it should fall back to the `en-us` template as the default since this would break the entire login functionality for end users.

**Review by solocommand — APPROVED** [review]

**Pull request merged**